auslib/db.py

```diff
@@ -713,17 +714,17 @@ def _buildIDMatchesRule(self, ruleBuildID, queryBuildID):
             return True
         return string_compare(queryBuildID, ruleBuildID)
 
-    def _osVersionMatchesRule(self, ruleOsVersion, queryOsVersion):
-        """Decides whether an osVersion from a rule matches an incoming one.
-           osVersion columns in a rule may specify multiple OS versions,
-           delimited by a comma. Once split we do simple substring matching
-           against the query's OS version. Unlike versions and channels, we
-           assume each OS version is a substring of what will be in the query,
-           thus we don't need to support globbing."""
-        if ruleOsVersion is None:
+    def _csvMatchesRule(self, ruleString, queryString):
+        """Decides whether a column from a rule matches an incoming one.
+           Some columns in a rule may specify multiple values delimited by a
+           comma. Once split we do simple substring matching against the query
+           string. Unlike versions and channels, we assume these columns
+           contain a substring of what will be in the query, thus we don't
+           need to support globbing."""
+        if ruleString is None:
             return True
-        for os in ruleOsVersion.split(','):
-            if os in queryOsVersion:
+        for os in ruleString.split(','):
+            if os in queryString:
```

Comment:
Nit, the os here looks like the last leftover from the renaming, so calling it something else would fit better.